### ISSUE TITLE
Symfony 4 support

### ISSUE THREAD

**Issue by samuelvi** [BeSimple/BeSimpleI18nRoutingBundle#109]
Hi, are you going to support Symfony 4? thanks!

**Comment by jsamouh** [comment]
any update ?

**Comment by remmel** [comment]
Not sure it will be needed as Symfony 4.1 will handle route translation natively : https://symfony.com/blog/new-in-symfony-4-1-internationalized-routing

**Comment by alcohol** [comment]
@remmel yeah but that means people/organizations currently using this Bundle will need to rewrite all their routing configuration (and that can be quite a large task depending on your application..).
